### ISSUE TITLE
Remove repeated names

### ISSUE THREAD

**Issue by hdliv** [phylotastic/phylotastic-portal#84]
When scraping a page with the portal (https://en.wikipedia.org/wiki/Elapidae).  several of the 'names' of 'entities' are repeated 2-3x times.  can we just list one?

**Comment by hdliv** [comment]
<img width="769" alt="screen shot 2016-04-28 at 14 58 45" src="https://cloud.githubusercontent.com/assets/6361361/14897938/3d8f143c-0d52-11e6-8a49-90de385fc059.png">

**Comment by ducvan0212** [comment]
I think uniquifying resolved names should be implemented inside @abusalehmdtayeen service. I can do uniquifying names in portal but it is not a sound design choice because what if we create another application for desktop, for example then developer for desktop app will have to do uniquifying things.

**Comment by abusalehmdtayeen** [comment]
@hdliv @ducvan0212 : Actually, I have noticed that this is a problem from the new GNRD service which returns these repeated names. This problem did not exist in the old version. I think this problem should be resolved from the root service which is GNRD.

**Comment by hdliv** [comment]
thanks @abusalehmdtayeen .  Let us know when this is resolved

**Comment by hdliv** [comment]
@abusalehmdtayeen has this been resolved?

**Comment by abusalehmdtayeen** [comment]
@hdliv : No, this has not been resolved yet. This must be done on Dima's end since it is a problem of GNRD itself. I think we should remind him in the next meeting.

**Comment by hdliv** [comment]
Thanks.  :)

**Comment by arlin** [comment]
I will assign this to @dimus.   Can you remove redundancy from scraped lists ?

**Comment by abusalehmdtayeen** [comment]
@hdliv @ducvan0212 , I have modified our wrapper GNRD web service to remove duplicates from the scientific names list. Can you check whether this feature is working or not?

**Comment by hdliv** [comment]
@abusalehmdtayeen @ducvan0212 There are still duplicates

**Comment by hdliv** [comment]
https://en.wikipedia.org/wiki/Elapidae
![screen shot 2016-09-16 at 09 56 50](https://cloud.githubusercontent.com/assets/6361361/18588331/0022c4fc-7bf4-11e6-8bb2-66a6bd41c176.png)

![screen shot 2016-09-16 at 09 56 23](https://cloud.githubusercontent.com/assets/6361361/18588340/072c5538-7bf4-11e6-8846-b24a33bf3fd2.png)

**Comment by hdliv** [comment]
https://en.wikipedia.org/wiki/Ant

![screen shot 2016-09-16 at 09 53 49](https://cloud.githubusercontent.com/assets/6361361/18588364/22866576-7bf4-11e6-8d93-266b3e09d923.png)
![screen shot 2016-09-16 at 09 54 49](https://cloud.githubusercontent.com/assets/6361361/18588363/228648de-7bf4-11e6-8ac0-ed41329bd55e.png)
![screen shot 2016-09-16 at 09 55 12](https://cloud.githubusercontent.com/assets/6361361/18588362/228647c6-7bf4-11e6-9816-0c9f4cd966d2.png)

**Comment by abusalehmdtayeen** [comment]
@hdliv , I checked my web service results for the URLs you provided. There are no duplicates. If you are using old list, duplicates may show up. You need to scrap the web pages from the beginning and create the lists again. Also there may be some caching involved in the portal end. This causes the old result to show up. Contact with @ducvan0212 about this issue. From now on the back end web service is capable of removing duplicate names.

**Comment by hdliv** [comment]
@abusalehmdtayeen OK, because I used new lists and not old ones.  I see the same with this wiki page: https://en.wikipedia.org/wiki/Algae

<img width="623" alt="screen shot 2016-09-16 at 13 10 23" src="https://cloud.githubusercontent.com/assets/6361361/18594667/170155e2-7c0f-11e6-8017-95386beee95d.png">

I will see wih @ducvan0212 about it
